vagrant/scripts/build_rust_rpms.sh

```diff
@@ -0,0 +1,9 @@
+yum install -y openssl-devel
+yum install -y cargo rpm-build
+
+[ -f /root/.cargo/bin/sccache ] && echo "sccache already installed. Skipping." || cargo install sccache
+
+rm -rf /integrated-manager-for-lustre/_topdir
```

Comment:
Why?

Reply:
I figured I should clear out the _topdir between each run. Does it not make sense to do this?